metaworld/envs/mujoco/sawyer_xyz/sawyer_reach_push_pick_place_v2.py

```diff
@@ -0,0 +1,326 @@
+import numpy as np
```

Comment:
can you split the v2 versions of these environments into 3 independent classes?

Reply:
Should I split the xml file as well?

Reply:
I trust you judgement